crates/router/src/connector/forte/transformers.rs

```diff
@@ -0,0 +1,263 @@
+    pub locality: String,
+    pub region: String,
+    pub country: String,
+    #[serde(rename = "postal_code")]
+    pub postal_code: String,
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Links {
+    pub disputes: String,
+    pub settlements: String,
+    #[serde(rename = "self")]
+    pub self_field: String,
+}
+
+//TransactionId Types end
+//Types End
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for FortePaymentsRequest  {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self,Self::Error> {
+
+        match item.request.payment_method_data {
+            api::PaymentMethod::Card(ref ccard) => {
+                let request  =  FortePaymentsRequest{
+                    billing_address : BillingAddress { first_name: ccard.card_holder_name.peek().clone(), last_name: ccard.card_holder_name.peek().clone() },
+                    card: Card {
+                        card_type               : String::from("visa"),
+                        name_on_card            : ccard.card_holder_name.peek().clone(),
+                        account_number          : ccard.card_number.peek().clone(),
+                        expire_month            : ccard.card_exp_month.peek().clone(),
+                        expire_year             : ccard.card_exp_year.peek().clone(),
+                        card_verification_value : ccard.card_cvc.peek().clone(),
+                    },
+                    authorization_amount: item.request.amount as f64,
+                    subtotal_amount: item.request.amount as f64,
+                };
+                Ok(request)
+            }
+            _ => Err(
+                errors::ConnectorError::NotImplemented("Current Payment Method".to_string()).into(),
+            ),
+    }
+}
+}
+
+//TODO: Fill the struct with respective fields
+// Auth Struct
+pub struct ForteAuthType {
+    pub(super) api_key: String
+}
+
+impl TryFrom<&types::ConnectorAuthType> for ForteAuthType  {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(_auth_type: &types::ConnectorAuthType) -> Result<Self, Self::Error> {
+        todo!()
+    }
+}
+// PaymentsResponse
+//TODO: Append the remaining status flags
+#[derive(Debug, Clone, Default, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum FortePaymentStatus {
+    Succeeded,
+    Failed,
+    #[default]
+    Processing,
+}
+
+impl<F,T> TryFrom<types::ResponseRouterData<F, FortePaymentsResponse, T, types::PaymentsResponseData>> for types::RouterData<F, T, types::PaymentsResponseData> {
+    type Error = error_stack::Report<errors::ParsingError>;
+    fn try_from(item: types::ResponseRouterData<F, FortePaymentsResponse, T, types::PaymentsResponseData>) -> Result<Self,Self::Error> {
+        Ok(Self {
+            status: enums::AttemptStatus::Authorized,
```

Comment:
why is the status hardcoded, how will you handle failure and other scenarios

Reply:
proper enum description is not available  in Forte documentation. Removed hardcoding and added mapping for available statuses.